x-element.js

```diff
@@ -172,6 +172,12 @@ export default class XElement extends HTMLElement {
     XElement.#connectHost(this);
   }
 
+  // TODO: #254: Uncomment once we leverage “moveBefore”.
+  // /**
+  //  * Extends HTMLElement.prototype.connectedMoveCallback.
+  //  */
+  // connectedMoveCallback() {}
```

Comment:
We’ll eventually want to add this to the base element for completeness / self-documentation, similar to other lifecycle methods.